src/components/Dailog.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="main_container">
+    <div class="container" v-for="issue in issues" :key="issue.id">
+      <v-alert v-show="showAlert(issue)" class="dailog" type="error"
+        >{{ issue.bName }} to be returned on {{ issue.dueDate }}
```

Comment:
is this field bName coming from a pre defined url or created by you in local store. If its in local store then give it a readable name like bookName.